src/_compress.pyx

```diff
@@ -18,7 +18,9 @@ cimport libc.stdio
 cimport openmp
 
 from libc.stdlib cimport malloc, free, realloc
-from libc.stdint cimport uint8_t, uint32_t
+ctypedef unsigned char  uint8_t
+ctypedef unsigned int   uint32_t
```

Comment:
`int` is compiler specific, so can be 2 or 4 bytes (or maybe 8?). Perhaps this is fine, I just thought `long int` is nearly always 32-bit (and `long long int` is 64 bits), maybe this isn't he case though...